pil-analyzer/src/statement_processor.rs

```diff
@@ -420,6 +436,36 @@ where
         }
     }
 
+    fn process_struct_declaration(
+        &mut self,
+        symbol: Symbol,
+        struct_decl: StructDeclaration<parsed::Expression>,
+    ) -> Vec<PILItem> {
+        let type_vars = struct_decl.type_vars.vars().collect();
+        let fields = struct_decl
+            .fields
+            .into_iter()
+            .map(|v| NamedType {
+                name: v.name.to_string(),
```

Comment:
Can't you move here? Please use `.clone()` instead of `to_string()` if you are just cloning.